lib/ledger_sync/resources/vendor.rb

```diff
@@ -1,12 +1,17 @@
+# frozen_string_literal: true
+
 module LedgerSync
   class Vendor < LedgerSync::Resource
+    attribute :company_name, type: Type::String
     attribute :email, type: Type::String
+    attribute :display_name, type: Type::String
     attribute :first_name, type: Type::String
     attribute :last_name, type: Type::String
-    attribute :display_name, type: Type::String
+    attribute :phone_number, type: Type::String
+    references_one :subsidiary, to: Subsidiary
 
     def name
-      "#{display_name}"
+      display_name.to_s
```

Comment:
Why do you need `to_s`

Reply:
Oh is it because it's a class, `Type::String`?

Reply:
I actually don't know why it was a string to begin with.  I believe my rubocop linter just changed this.  Tests pass without it, though I will leave for now.

@jozefvaclavik did you just keep it because it used to be `"#{first_name} #{last_name}"`?  https://github.com/LedgerSync/ledger_sync/blame/4d4ad328fd3af712a88d0e7f07192e0cb349fcbb/lib/ledger_sync/resources/vendor.rb